### PR TITLE
Remove v1 authentication

### PR DESCRIPTION
**Pull request by tomas-stefano**
[Trello](https://trello.com/c/DaEXYM24/109-switch-to-new-auth-method-deprecate-old-auth-method-4-4)

## Description

Removes the v1 Authentication when send9ing data to file store, data store and submitter.

## Caveats

The service token were being used for two purposes:

1) Sending in the v1 authentication
2) Generating the base token and using to validate crsf alongside with the user session id.

So I removed the service token from the step 1. Because it is not the scope of this work I decided to leave the SERVICE_TOKEN as it for the second step but at least on the code I decided to rename to CSRF_TOKEN which makes more sense to me.

### CODE REVIEW

File: lib/server/server-middleware.js [hunk 1]
```diff
@@ -45,6 +45,8 @@ const configureMiddleware = (options = {}) => {
     GA_TRACKING_ID
   } = options
 
+  const CSRF_TOKEN = SERVICE_TOKEN
```
Comment (sequencemedialimited) [inline lib/server/server-middleware.js]: There's no need to re-assign the value of a variable just for the sake of the name when you're also destructuring:

```
const {
    serviceSources,
    ENV,
    PLATFORM_ENV,
    EDITABLE,
    SERVICE_SECRET,
    SERVICE_TOKEN: CSRF_TOKEN,
    FQD,
    USERNAME,
    PASSWORD,
    REALM,
    ASSET_PATH,
    ASSET_SRC_PATH,
    SENTRY_DSN,
    GA_TRACKING_ID
  } = options
```
This destructuring statement achieves the same thing

(And I'm not sure there's anything _gained_ by renaming the variable, anyway)

**Comment by sequencemedialimited** [comment]
Otherwise, fine and I'll leave it up to you to decide whether to change anything

File: lib/server/server-middleware.js (hunk 1 above)
Comment (tomas-stefano) [reply lib/server/server-middleware.js]: Yes, you're right. I did that in order to sound clearer what it was used for, but you're alright. I am happy to change.

Comment (tomas-stefano) [reply lib/server/server-middleware.js]: Done! ♻️

File: README.md [hunk 2]
```diff
@@ -52,9 +52,7 @@ export USER_DATASTORE_URL=http://localhost:44444
 export USER_FILESTORE_URL=http://localhost:44445
 export SUBMITTER_URL=http://localhost:44446
 export SERVICE_SECRET=sekrit
-export SERVICE_TOKEN=token
 export SERVICE_SLUG=slug
-export MODULE__ministryofjustice_fb_components_core='/path/to/your/form-builder/fb-components-core'
```
Comment (asmega) [inline README.md]: i think this is still being used?

**Pull request merged**